python/neuroglancer/pipeline/storage.py

```diff
@@ -107,6 +117,50 @@ def get_file(self, file_path):
             content = self._maybe_uncompress(content)
         return content
 
+    def get_files(self, file_paths):
+        """
+        returns a list of files faster by using threads
+        """
+
+        file_paths = deque(file_paths)
+
+        results = []
+
+        def store_result(path, result, error):
```

Comment:
```
@staticmethod
def store_result(path, result, error):
```
and moving it to the class  I think will make get_files a little bit shorter and easier to understand.